index.html

```diff
@@ -47,6 +52,217 @@ <h4>I am a Software Developer</h4>
+                <p class="third_p">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups
+                    required.
+                    Has been the<br>industry's standard
+                </p>
+            </div>
+            <div>
+                <ul class="middle-button2">
+                    <li class="css-button2">HTML</li>
+                    <li class="html-button2">BOOTSTRAP</li>
+                    <li class="bootstrap-button2">RUBY</li>
+                </ul>
+            </div>
+            <div>
+                <button class="project-button2">SEE PROJECT</button>
+            </div>
+        </section>
+
+        <section class="grid-container2">
+            <div>
+                <h1 class="third_h1">Professional Art<br>Printing Data</h1>
+                <p class="third_p">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups
+                    required.
+                    Has been the<br>industry's standard
+                </p>
+            </div>
+            <div>
+                <ul class="middle-button2">
+                    <li class="css-button2">HTML</li>
+                    <li class="html-button2">BOOTSTRAP</li>
+                    <li class="bootstrap-button2">RUBY</li>
+                </ul>
+            </div>
+            <div>
+                <button class="project-button2">SEE PROJECT</button>
+            </div>
+        </section>
+
+        <section class="grid-container2">
+            <div>
+                <h1 class="third_h1">Professional Art<br>Printing Data</h1>
+                <p class="third_p">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups
+                    required.
+                    Has been the<br>industry's standard
+                </p>
+            </div>
+            <div>
+                <ul class="middle-button2">
+                    <li class="css-button2">HTML</li>
+                    <li class="html-button2">BOOTSTRAP</li>
+                    <li class="bootstrap-button2">RUBY</li>
+                </ul>
+            </div>
+            <div>
+                <button class="project-button2">SEE PROJECT</button>
+            </div>
+        </section>
+
+        <section class="about-me">
+            <div>
+                <h1 class="about-section-h1">About me</h1>
+                <p class="about-section-paragraph">Hello. I am software developer. I can help you build a product,
+                    feature or website. Look through some
+                    of my work and experience! If you like what you see and have a project you need coded, do not
+                    hesitate to contact me</p>
+            </div>
+            <div>
+                <button class="about-section-button">Get My Resume</button>
+            </div>
+        </section>
+
+        <section class="aboutme-container2">
+            <div class="line-1"></div>
```

Comment:
- [ ] Kindly let's use a single `<section>` tag for the `About me` section as well instead of having two separate `<section>` tags for the `About me`. 👍